From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 12_maldetect-cron.daily-documentroots

---
 cron.daily | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

--- a/cron.daily
+++ b/cron.daily
@@ -6,6 +6,10 @@
 	. /etc/maldetect/maldetect.conf.cron
 fi
 find=`which find 2> /dev/null`
+if [ ! "${STDDOCROOT}" ]; then
+	STDDOCROOT="/var/www/,/srv/www/"
+fi
+
 if [ "$find" ]; then
 	# prune any quarantine/session/tmp data older than 7 days
 	tmpdirs="/tmp/maldetect /var/lib/maldetect/sess /var/lib/maldetect/quarantine /var/lib/maldetect/pub/"
@@ -55,6 +59,6 @@
 		/usr/bin/maldet -b -r /var/www/?/data/,/home/?/data/ 1 >> /dev/null 2>&1
 	else
 		# cpanel, interworx and other standard home/user/public_html setups
-	        /usr/bin/maldet -b -r /home?/?/public_html/,/var/www/html/,/usr/local/apache/htdocs/ 1 >> /dev/null 2>&1
+	        /usr/bin/maldet -b -r /home?/?/public_html/,/var/www/html/,${STDDOCROOT} 1 >> /dev/null 2>&1
 	fi
 fi
