From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 12_maldetect-cron.daily-documentroots

---
 cron.daily | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

--- a/cron.daily
+++ b/cron.daily
@@ -31,6 +31,10 @@
 	scan_days=1
 fi
 
+if [ ! "${STDDOCROOT}" ]; then
+	STDDOCROOT="/var/www/,/srv/www/"
+fi
+
 if [ "$find" ]; then
 	# prune any quarantine/session/tmp data older than 7 days
 	tmpdirs="$tmpdir $varlibpath/sess $varlibpath/quarantine $varlibpath/pub"
@@ -86,7 +90,7 @@
                 $inspath/maldet -b -r /home/?/web/?/public_html/,/home/?/web/?/public_shtml/,/home/?/tmp/,/home/?/web/?/private/ $scan_days >> /dev/null 2>&1
 	else
 		# cpanel, interworx and other standard home/user/public_html setups
-	        $inspath/maldet -b -r /home?/?/public_html/,/var/www/html/,/usr/local/apache/htdocs/ $scan_days >> /dev/null 2>&1
+	        $inspath/maldet -b -r /home?/?/public_html/,/var/www/html/,${STDDOCROOT} $scan_days >> /dev/null 2>&1
 	fi
 fi
 
