From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 12_maldetect-cron.daily-documentroots

---
 cron.daily | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

--- a/cron.daily
+++ b/cron.daily
@@ -4,6 +4,10 @@
 # sleep for random value to give upstream a bit of headroom
 sleep $(echo $RANDOM | cut -c1-3) >> /dev/null 2>&1
 
+if [ ! "${STDDOCROOT}" ]; then
+	STDDOCROOT="/var/www/html /usr/local/apache/htdocs /var/www/ /srv/www/"
+fi
+
 # clear quarantine/session/tmp data every 14 days
 /usr/sbin/tmpwatch 336 /tmp/maldetect >> /dev/null 2>&1
 /usr/sbin/tmpwatch 336 /var/lib/maldetect/sess >> /dev/null 2>&1
@@ -36,10 +40,9 @@
 	fi
 
 	# scan default apache docroot paths
-	if [ -d "/var/www/html" ]; then
-		/usr/bin/maldet -b -r /var/www/html 2 >> /dev/null 2>&1
-	fi
-	if [ -d "/usr/local/apache/htdocs" ]; then
-		/usr/bin/maldet -b -r /usr/local/apache/htdocs 2 >> /dev/null 2>&1
-	fi
+	for DIR in ${STDDOCROOT}; do
+		if [ -d "${DIR}" ]; then
+			 /usr/bin/maldet -b -r ${DIR} 2 >> /dev/null 2>&1
+		fi
+	done
 fi
