From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 12_maldetect-cron.daily-documentroots

---
 cron.daily | 15 +++++++++------
 1 file changed, 9 insertions(+), 6 deletions(-)

--- a/cron.daily
+++ b/cron.daily
@@ -23,6 +23,10 @@
 fi
 
 find=`which find 2> /dev/null`
+if [ ! "${STDDOCROOT}" ]; then
+	STDDOCROOT="/var/www/,/srv/www/"
+fi
+
 if [ "$find" ]; then
 	# prune any quarantine/session/tmp data older than 7 days
 	tmpdirs="/tmp/maldetect $var_path/sess $var_path/quarantine $var_path/pub"
@@ -78,7 +82,7 @@
                 $install_path/maldet -b -r /home/?/web/?/public_html/,/home/?/web/?/public_shtml/,/home/?/tmp/,/home/?/web/?/private/ $scan_days >> /dev/null 2>&1
 	else
 		# cpanel, interworx and other standard home/user/public_html setups
-	        $install_path/maldet -b -r /home?/?/public_html/,/var/www/html/,/usr/local/apache/htdocs/ $scan_days >> /dev/null 2>&1
+	        $install_path/maldet -b -r /home?/?/public_html/,/var/www/html/,${STDDOCROOT} $scan_days >> /dev/null 2>&1
 	fi
 fi
 
