From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 13_maldetect-cron.daily-default

---
 cron.daily | 4 ++++
 1 file changed, 4 insertions(+)

--- a/cron.daily
+++ b/cron.daily
@@ -4,6 +4,10 @@
 # sleep for random value to give upstream a bit of headroom
 sleep $(echo $RANDOM | cut -c1-3) >> /dev/null 2>&1
 
+if [ -r "/etc/default/maldetect" ]; then
+	source /etc/default/maldetect
+fi
+
 if [ ! "${STDDOCROOT}" ]; then
 	STDDOCROOT="/var/www/html /usr/local/apache/htdocs /var/www/ /srv/www/"
 fi
