From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 13_maldetect-cron.daily-default

---
 cron.daily | 4 ++++
 1 file changed, 4 insertions(+)

--- a/cron.daily
+++ b/cron.daily
@@ -23,6 +23,10 @@
 fi
 
 find=`which find 2> /dev/null`
+if [ -r "/etc/default/maldetect" ]; then
+	source /etc/default/maldetect
+fi
+
 if [ ! "${STDDOCROOT}" ]; then
 	STDDOCROOT="/var/www/,/srv/www/"
 fi
