From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 15_maldetect-no-autoupdate

---
 files/maldet | 81 +-----------------------------------------------------------
 1 file changed, 1 insertion(+), 80 deletions(-)

--- a/files/maldet
+++ b/files/maldet
@@ -236,10 +236,6 @@
 				header
 				purge
 			;;
-			-d|--update-ver|--update-version)
-				header
-				lmdup
-			;;
 			-u|--update|--update-sigs)
 				shift
 				if [ ! "$1" == "1" ]; then
--- a/files/internals/functions
+++ b/files/internals/functions
@@ -296,7 +296,7 @@
       [-f|--file-list PATH] [-i|--include-regex] [-x|--exclude-regex]
       [-b|--background] [-m|--monitor] [-k|--kill-monitor] [-c|--checkout]
       [-q|--quarantine] [-s|--restore] [-n|--clean] [-l|--log] [-e|--report]
-      [-u|--update-sigs] [-d|--update-ver]
+      [-u|--update-sigs]
 EOF
 }
 
@@ -311,9 +311,6 @@
     -u, --update-sigs
        Update malware detection signatures from rfxn.com
 
-    -d, --update-ver
-       Update the installed version from rfxn.com
-
     -f, --file-list
        Scan files or paths defined in line spaced file
        e.g: maldet -f /root/scan_file_list
@@ -1696,95 +1693,6 @@
 	fi
 }
 
-
-lmdup() {
-	tmpwd="$tmpdir/.lmdup.$RANDOM.$$"
-	upstreamver="$tmpwd/.lmdup_vercheck.$$"
-	mkdir -p $tmpwd ; chmod 700 $tmpwd
-	
-	eout "{update} checking for available updates..." 1
-	
-	$wget --referer="$lmd_referer" -q -T$wget_timeout -t$wget_retries "$lmd_version_url" -O $upstreamver >> /dev/null 2>&1
-	if [ -s "$upstreamver" ]; then
-		installedver=`echo $ver | tr -d '.'`
-		if [ "$(echo $installedver | wc -L)" -eq "2" ]; then
-			installedver="${installedver}0"
-		fi
-		upstreamver_readable=`cat $upstreamver`
-		upstreamver=`cat $upstreamver | tr -d '.'`
-		if [ "$upstreamver" -gt "$installedver" ]; then
-			eout "{update} new version $upstreamver_readable found, updating..." 1
-			doupdate=1
-			elif [ "$autoupdate_version_hashed" == "1" ]; then
-			eout "{update} hashing install files and checking against server..." 1
-			$md5sum $inspath/maldet $intfunc | awk '{print$1}' | tr '\n' ' ' | tr -d ' ' > $lmd_hash_file
-			upstreamhash="$tmpwd/.lmdup_hashcheck$$"
-			$wget --referer="$lmd_referer" -q -T$wget_timeout -t$wget_retries "$lmd_hash_url" -O $upstreamhash >> /dev/null 2>&1
-			if [ -s "$upstreamhash" ]; then
-				installed_hash=`cat $lmd_hash_file`
-				current_hash=`cat $upstreamhash`
-				if [ ! "$installed_hash" == "$current_hash" ]; then
-					eout "{update} version check shows latest but hash check failed, forcing update..." 1
-					doupdate=1
-				else
-					eout "{update} latest version already installed." 1
-				fi
-			else
-				eout "{update} could not download upstream hash file ($lmd_hash_url), please try again later." 1
-				cd $inspath ; rm -rf $tmpwd
-				clean_exit
-				exit 1
-			fi
-		else
-			eout "{update} no updates available, latest version already installed." 1
-		fi
-	else
-		eout "{update} could not download version file from server, please try again later." 1
-		cd $inspath ; rm -rf $tmpwd
-		clean_exit
-		exit 1
-	fi
-	if [ "$doupdate" ]; then
-		cd $tmpwd/
-		$wget --referer="$lmd_referer" -q -T$wget_timeout -t$wget_retries "http://cdn.rfxn.com/downloads/maldetect-current.tar.gz" -O "$tmpwd/maldetect-current.tar.gz"
-		$wget --referer="$lmd_referer" -q -T$wget_timeout -t$wget_retries "http://cdn.rfxn.com/downloads/maldetect-current.tar.gz.md5" -O "$tmpwd/maldetect-current.tar.gz.md5"
-		if [ -s "$tmpwd/maldetect-current.tar.gz.md5" ] && [ -s "$tmpwd/maldetect-current.tar.gz" ]; then
-			upstream_md5=`cat $tmpwd/maldetect-current.tar.gz.md5 | awk '{print$1}'`
-			local_md5=`$md5sum $tmpwd/maldetect-current.tar.gz | awk '{print$1}'`
-			if [ ! "$upstream_md5" == "$local_md5" ]; then
-				eout "{update} unable to verify md5sum of maldetect-current.tar.gz, update failed!" 1
-				cd $inspath ; rm -rf $tmpwd
-				clean_exit
-				exit 1
-			else
-				eout "{update} verified md5sum of maldetect-current.tar.gz" 1
-			fi
-		else
-			eout "{update} could not download maldetect-current.tar.gz or .md5, please try again later." 1
-			cd $inspath ; rm -rf $tmpwd
-			clean_exit
-			exit 1
-		fi
-		if [ -s "$tmpwd/maldetect-current.tar.gz" ]; then
-			tar xfz maldetect-current.tar.gz
-			cd maldetect-*
-			chmod 750 install.sh
-			sh -c './install.sh' >> /dev/null 2>&1
-			cp -f $inspath.last/sigs/custom.* $sigdir/ 2> /dev/null
-			cp -f $inspath.last/clean/custom.* $inspath/clean/ 2> /dev/null
-			eout "{update} completed update v$ver => v$upstreamver_readable, running signature updates..." 1
-			$inspath/maldet --update 1
-			eout "{update} update and config import completed" 1
-		else
-			eout "{update} could not download maldetect-current.tar.gz, please try again later." 1
-			cd $inspath ; rm -rf $tmpwd
-			clean_exit
-			exit 1
-		fi
-	fi
-	cd $inspath ; rm -rf $tmpwd
-}
-
 sigup() {
 	eout "{sigup} performing signature update check..." 1
 	
