From: Jan Wagner <waja@cyconet.org>
Date: Thu, 12 Mar 2015 22:41:40 +0100
Subject: 15_maldetect-no-autoupdate

---
 files/maldet | 81 +-----------------------------------------------------------
 1 file changed, 1 insertion(+), 80 deletions(-)

--- a/files/internals/functions
+++ b/files/internals/functions
@@ -438,7 +438,7 @@
       [-f|--file-list PATH] [-i|--include-regex] [-x|--exclude-regex]
       [-b|--background] [-m|--monitor] [-k|--kill-monitor] [-c|--checkout]
       [-q|--quarantine] [-s|--restore] [-n|--clean] [-l|--log] [-e|--report]
-      [-u|--update-sigs] [-d|--update-ver]
+      [-u|--update-sigs]
 EOF
 }
 
@@ -453,9 +453,6 @@
     -u, --update-sigs [--force]
        Update malware detection signatures from rfxn.com
 
-    -d, --update-ver [--force]
-       Update the installed version from rfxn.com
-
     -f, --file-list
        Scan files or paths defined in line spaced file
        e.g: maldet -f /root/scan_file_list
@@ -2149,111 +2146,6 @@
 	fi
 }
 
-
-lmdup() {
-	tmpwd="$tmpdir/.lmdup.$RANDOM.$$"
-	upstreamver="$tmpwd/.lmdup_vercheck.$$"
-	mkdir -p $tmpwd ; chmod 700 $tmpwd
-
-	if [ "$lmdup_beta" ]; then
-		lmd_hash_url="${lmd_hash_url}.beta"
-		lmd_version_url="${lmd_version_url}.beta"
-		lmd_current_tgzfile="maldetect-beta.tar.gz"
-	fi
-	
-	eout "{update} checking for available updates..." 1
-        get_remote_file "$lmd_version_url" "update" "1"
-	upstreamver="$return_file"	
-	if [ -s "$upstreamver" ]; then
-		installedver=`echo $ver | tr -d '.'`
-		if [ "$(echo $installedver | wc -L)" -eq "2" ]; then
-			installedver="${installedver}0"
-		fi
-		upstreamver_readable=`cat $upstreamver`
-		upstreamver=`cat $upstreamver | tr -d '.'`
-		if [ "$(echo $upstreamver | wc -L)" -eq "2" ]; then
-			upstreamver="${upstreamver}0"
-		fi
-		if [ "$upstreamver" -gt "$installedver" ]; then
-			eout "{update} new version $upstreamver_readable found, updating..." 1
-			doupdate=1
-		elif [ "$lmdup_force" ]; then
-			eout "{update} version update with --force requested, updating..." 1
-			doupdate=1
-		elif [ "$autoupdate_version_hashed" == "1" ]; then
-			eout "{update} hashing install files and checking against server..." 1
-			eval $md5sum $inspath/maldet $intfunc | awk '{print$1}' | tr '\n' ' ' | tr -d ' ' > $lmd_hash_file
-			upstreamhash="$tmpwd/.lmdup_hashcheck$$"
-		        get_remote_file "$lmd_hash_url" "update" "1"
-			upstreamhash="$return_file"
-			if [ -s "$upstreamhash" ]; then
-				installed_hash=`cat $lmd_hash_file`
-				current_hash=`cat $upstreamhash`
-				if [ ! "$installed_hash" == "$current_hash" ]; then
-					eout "{update} version check shows latest but hash check failed, forcing update..." 1
-					doupdate=1
-				else
-					eout "{update} latest version already installed." 1
-				fi
-			else
-				eout "{update} could not download upstream hash file ($lmd_hash_url), please try again later." 1
-				cd $inspath ; rm -rf $tmpwd
-				clean_exit
-				exit 1
-			fi
-		else
-			eout "{update} no updates available, latest version already installed." 1
-		fi
-	else
-		eout "{update} could not download version file from server, please try again later." 1
-		cd $inspath ; rm -rf $tmpwd
-		clean_exit
-		exit 1
-	fi
-	if [ "$doupdate" ]; then
-		cd $tmpwd/
-
-                get_remote_file "${lmd_current_tgzbase_url}/${lmd_current_tgzfile}" "update" "1" "$tmpwd/${lmd_current_tgzfile}"
-                get_remote_file "${lmd_current_tgzbase_url}/${lmd_current_tgzfile}.md5" "update" "1" "$tmpwd/${lmd_current_tgzfile}.md5"
-
-		if [ -s "$tmpwd/${lmd_current_tgzfile}.md5" ] && [ -s "$tmpwd/${lmd_current_tgzfile}" ]; then
-			upstream_md5=`cat $tmpwd/${lmd_current_tgzfile}.md5 | awk '{print$1}'`
-			local_md5=`eval $md5sum $tmpwd/${lmd_current_tgzfile} | awk '{print$1}'`
-			if [ ! "$upstream_md5" == "$local_md5" ]; then
-				eout "{update} unable to verify md5sum of ${lmd_current_tgzfile}, update failed!" 1
-				cd $inspath ; rm -rf $tmpwd
-				clean_exit
-				exit 1
-			else
-				eout "{update} verified md5sum of ${lmd_current_tgzfile}" 1
-			fi
-		else
-			eout "{update} could not download ${lmd_current_tgzfile} or .md5, please try again later." 1
-			cd $inspath ; rm -rf $tmpwd
-			clean_exit
-			exit 1
-		fi
-		if [ -s "$tmpwd/${lmd_current_tgzfile}" ]; then
-			tar xfz ${lmd_current_tgzfile}
-			rm -f ${lmd_current_tgzfile} ${lmd_current_tgzfile}.md5
-			cd maldetect-${upstreamver_readable}
-			chmod 750 install.sh
-			sh -c './install.sh' >> /dev/null 2>&1
-			cp -f $inspath.last/sigs/custom.* $sigdir/ 2> /dev/null
-			cp -f $inspath.last/clean/custom.* $inspath/clean/ 2> /dev/null
-			eout "{update} completed update v$ver ${installed_hash:0:6} => v$upstreamver_readable ${upstream_md5:0:6}, running signature updates..." 1
-			$inspath/maldet --update 1
-			eout "{update} update and config import completed" 1
-		else
-			eout "{update} could not download ${lmd_current_tgzfile}, please try again later." 1
-			cd $inspath ; rm -rf $tmpwd
-			clean_exit
-			exit 1
-		fi
-	fi
-	cd $inspath ; rm -rf $tmpwd
-}
-
 sigup() {
 	eout "{sigup} performing signature update check..." 1
 	
