From 8fe097688ff45dc86044605f92ad57b4b223f2c3 Mon Sep 17 00:00:00 2001
From: Jan Wagner <waja@cyconet.org>
Date: Mon, 21 Sep 2015 17:37:24 +0200
Subject: [PATCH] Removing absolut path of 'pidof'

---
 files/hookscan.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/files/hookscan.sh b/files/hookscan.sh
index 2718086..74a2e57 100755
--- a/files/hookscan.sh
+++ b/files/hookscan.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 file="$1"
-isclamd=`/sbin/pidof clamd 2> /dev/null`
+isclamd=`pidof clamd 2> /dev/null`
 clamdloc=`which clamdscan 2> /dev/null`
 if [ "$isclamd" ] && [ -f "$clamdloc" ]; then
 	clamd_scan=1
