From 23f726b1154aa91a93046ef032fb05270a3a6754 Mon Sep 17 00:00:00 2001
From: Jan Wagner <waja@cyconet.org>
Date: Tue, 6 Oct 2015 14:06:23 +0200
Subject: [PATCH] Replace remaining temporary dirs with

---
 files/internals/functions | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

--- a/files/internals/functions
+++ b/files/internals/functions
@@ -1639,11 +1639,11 @@
 }
 
 gensigs() {
-	runtime_ndb="$inspath/tmp/.runtime.user.$$.ndb"
-	runtime_hdb="$inspath/tmp/.runtime.user.$$.hdb"
+	runtime_ndb="$tmpdir/.runtime.user.$$.ndb"
+	runtime_hdb="$tmpdir/.runtime.user.$$.hdb"
 	ln -fs $runtime_ndb $inspath/sigs/lmd.user.ndb
 	ln -fs $runtime_hdb $inspath/sigs/lmd.user.hdb
-	runtime_hexstrings="$inspath/tmp/.runtime.hexsigs.$$"
+	runtime_hexstrings="$tmpdir/.runtime.hexsigs.$$"
 	if [ -s "$sig_cust_hex_file" ]; then
 		cat "$sig_hex_file" "$sig_cust_hex_file" > $runtime_hexstrings
 	else
