From 6b60d2b0565577dc63e247b77c944b832f473297 Mon Sep 17 00:00:00 2001
From: Jan Wagner <waja@cyconet.org>
Date: Tue, 6 Oct 2015 15:20:06 +0200
Subject: [PATCH] Replace remaining sigdirs dirs with

---
 files/internals/functions | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

--- a/files/internals/functions
+++ b/files/internals/functions
@@ -286,8 +286,8 @@
 clamav_linksigs() {
 	cpath="$1"
 	if [ -d "$cpath" ]; then
-		rm -f $cpath/rfxn.* ; cp -f $inspath/sigs/rfxn.ndb $inspath/sigs/rfxn.hdb $cpath/ 2> /dev/null
-		rm -f $cpath/lmd.user.* ; cp -f $inspath/sigs/lmd.user.ndb $inspath/sigs/lmd.user.hdb $cpath/ 2> /dev/null
+		rm -f $cpath/rfxn.* ; cp -f $sigdir/rfxn.ndb $sigdir/rfxn.hdb $cpath/ 2> /dev/null
+		rm -f $cpath/lmd.user.* ; cp -f $sigdir/lmd.user.ndb $sigdir/lmd.user.hdb $cpath/ 2> /dev/null
 	fi
 }
 
@@ -1641,8 +1641,8 @@
 gensigs() {
 	runtime_ndb="$tmpdir/.runtime.user.$$.ndb"
 	runtime_hdb="$tmpdir/.runtime.user.$$.hdb"
-	ln -fs $runtime_ndb $inspath/sigs/lmd.user.ndb
-	ln -fs $runtime_hdb $inspath/sigs/lmd.user.hdb
+	ln -fs $runtime_ndb $sigdir/lmd.user.ndb
+	ln -fs $runtime_hdb $sigdir/lmd.user.hdb
 	runtime_hexstrings="$tmpdir/.runtime.hexsigs.$$"
 	if [ -s "$sig_cust_hex_file" ]; then
 		cat "$sig_hex_file" "$sig_cust_hex_file" > $runtime_hexstrings
